src/main/hook-ti.ts

```diff
@@ -14,7 +14,9 @@ export const IOffsets = t.iface([], {
 	'playerCount': t.array('number'),
 	'playerAddrPtr': 'number',
 	'exiledPlayerId': t.array('number'),
-	'gameCode': t.array('number'),
+  'gameCode': t.array('number'),
+  'hostId': t.array('number'),
+  'clientId': t.array('number'),
 	'player': t.iface([], {
```

Comment:
indentation is different here

Reply:
Fixed in 4b3abc74fc22094cd7410a75cbfb8afbab9d6a8d